content/en/docs/getting-started/_index.md

```diff
@@ -34,10 +37,62 @@ apt-get install -y --no-install-recommends \
   unzip \
   uuid-runtime \
   python3 \
-  python3-setuptools
+  python3-setuptools \
+  python3-pip \
+  qemu-kvm \
+  qemu-system-x86 \
+  sgabios
+```
+
+### Arch Linux, Manjaro and other Pacman-based distributions
+
+Please run the following command to install the requirements:
+
+```shell-script
+pacman -Sy base-devel \
+  ncurses \
+  libyaml \
+  flex \
+  git \
+  wget \
+  socat \
+  bison \
+  unzip \
+  python \
+  python-setuptools \
+  python-pip \
+  qemu-system-x86
 ```
 
-To install kraft simply run:
+### Fedora and other dnf-based distributions
+
+Please run the following command to install the requirements:
+
+```shell-script
+dnf install -y \
+  @development-tools \
+  ncurses-devel \
+  libyaml \
+  flex \
+  git \
+  wget \
+  socat \
+  bison \
+  unzip \
+  libuuid \
+  python3 \
+  python3-setuptools \
+  python3-pip \
+  qemu-kvm \
+  qemu-system-x86 \
+  sgabios
+```
+
+**Note: Since on Fedora the default `yacc` is [`Berkley Yacc`](https://en.wikipedia.org/wiki/Berkeley_Yacc), not [`GNU Bison`](https://www.gnu.org/software/bison/), you will need to manually set the `YACC` env variable `export YACC=/usr/bin/bison`.**
```

Comment:
```suggestion
**Note: Since on Fedora the default `yacc` is [`Berkeley Yacc`](https://en.wikipedia.org/wiki/Berkeley_Yacc), not [`GNU Bison`](https://www.gnu.org/software/bison/), you will need to manually set the `YACC` env variable `export YACC=/usr/bin/bison`.**
```